.github/workflows/master.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           - '-Pspark-3.0 -Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.1.1 -Dspark.archive.name=spark-3.1.1-bin-hadoop2.7.tgz -Dmaven.plugin.scalatest.exclude.tags=org.apache.kyuubi.tags.DataLakeTest'
           - '-Pspark-3.1'
           - '-Pspark-3.1 -Dhadoop.binary.version=3.2'
+          - '-Pspark-3.2-snapshot -Dspark.archive.download.skip=true'
```

Comment:
`-pl :kyuubi-spark-sql-engine -Pspark-3.2-snapshot ` might be enough to verify API consistency with Spark

Reply:
ok, will fix in next PS.